kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/store/DbDataStore.java

```diff
@@ -182,6 +182,11 @@ private void init(int houseKeeperInterval, int purgeAge, int capacity) throws Ku
                 throw e;
             }
         }
+        
+        // Introduced in 3.0??.  create index that speeds up the process of deleting expired messages during housekeeping task
```

Comment:
Please also update the copyright header:

Copyright (c) 2011, 2017 Eurotech and/or its affiliates